packages/components/src/components/button-toggle-menu/button-toggle-menu.jsx

```diff
@@ -1,11 +1,12 @@
 import classNames       from 'classnames';
```

Comment:
`toggle-button` or `button-toggle` should be enough i think @oskar-binary @negar-binary ?

Reply:
@mustofa-binary I agree with you, but this was the name of this component in deriv-trader. is it ok to change it in both places?

Reply:
yes, there is only those 2 usages, can change the name 👍